blazor/datagrid/column-chooser.md

```diff
@@ -460,4 +460,53 @@ This can be demonstrated in the following sample:
 </style> 
 ```
 
-![Customize Column Chooser Dialog Size](./images/blazor-datagrid-customize-column-chooser-dialog.png)
\ No newline at end of file
+![Customize Column Chooser Dialog Size](./images/blazor-datagrid-customize-column-chooser-dialog.png)
+
+## Change default search operator of the column chooser
+
+The default operator for the column chooser search is **StartsWith**. It is possible to change the default operator using the [Operator](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Operator.html) property of the [GridColumnChooserSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumnChooserSettings.html) class.
```

Comment:
Column chooser seaech can be changed.
Default operator for search box in the column chooser

Reply:
Mention the column chooser settings. Also share the api settings